src/openforms/forms/api/serializers/form.py

```diff
@@ -317,6 +324,21 @@ def get_required_fields_with_asterisk(self, obj) -> bool:
         config = GlobalConfiguration.get_solo()
         return config.form_display_required_with_asterisk
 
+    def get_resume_link_lifetime(self, obj) -> int:
+        config = GlobalConfiguration.get_solo()
+        lifetime = obj.incomplete_submissions_removal_limit
+
+        if not lifetime:
+            lifetime = config.incomplete_submissions_removal_limit
+
+        if lifetime_all := (
+            obj.all_submissions_removal_limit or config.all_submissions_removal_limit
+        ):
+            if lifetime_all < lifetime:
+                lifetime = lifetime_all
```

Comment:
```suggestion
        lifetime = obj.incomplete_submissions_removal_limit or config.incomplete_submissions_removal_limit
        lifetime_all = obj.all_submissions_removal_limit or config.all_submissions_removal_limit
        lifetime = min(lifetime, lifetime_all)
```
let's make it flat :)

that will probably also address the coverage check failures